### DISCUSSION TITLE
What's the `_mapToAdd` for ?

### DISCUSSION THREAD

**Discussion by zxh19890103** [Leaflet/Leaflet#7938] [Q&A] [answered]
I found every layer would have two properties after mounted. One is `_map` which I know it's the instance of `L.Map` on which the layer being add. But still it has another property named `_mapToAdd` which is always the same thing to `_map`. My question is this: what does the property `_mapToAdd` use for?

**Comment by Falke-Design** [comment, marked as answer]
When the map is not ready, the layer can't added to the map, so it gets the `_mapToAdd` property. After the map is intialized the layer is added to the map (which uses `_mapToAdd`)

**Reply by zxh19890103** [reply]
Thanks.
